test/distributed/pipelining/test_stage.py

```diff
@@ -191,23 +189,6 @@ def _run_step(x):
         old_keys = mod.state_dict().keys()
         assert all(k in old_keys for k in submod_keys)
 
-        if self.rank == 0:
-            with self.assertRaisesRegex(PipeliningShapeError, "shape mismatch"):
-                _run_step(torch.randn(batch_size + 1, d_hid, device=self.device))
```

Comment:
how come the shape checking got disabled even for args? i though from reading the stage code change it would only change the validation of kwargs

Reply:
I disabled the check on args in the library code **when kwargs are present**. 
So I have to remove the `assertRaisesRegex` test here (otherwise it catches nothing).

This is the comment in library code:
```
            # TODO- need a mapping of kwarg to position in self.args_recv_info
            # Without it, we are not 100% sure how to match the args and
            # expected_args.
```